NMD.pm

```diff
@@ -151,8 +151,18 @@ sub variant_exon_check {
   # to check if the second to the last exon exists 
   if (defined($second_last_exon)){
     # this method has been changed because of the reverse strand consideration
-    my $coding_region_end = $second_last_exon->end;
-    my $diff_end = $coding_region_end - 51;
+    my $coding_region_end;
+    my $diff_end;
+    # need to account for strand as if it is in reverse strand it should be reading from the start 
+    # and adding 51 to find the variant 50 bases upstream in penultimate exon
+    if ($tr->strand == -1){
+      $coding_region_end = $second_last_exon->start;
+      $diff_end = $coding_region_start + 51;
```

Comment:
Hi Ola,
there a bit mishap here I think. With this logic the plugin will check from -
5'end    ....EE...ESEE...ESEE....ESEE....ES.... 3'end
                       ^.    ^           
but you want to search in -
5'end    ....EE...ESEE...ESEE....ESEE....ES.... 3'end
                              ^.    ^           

For example with variant `16	23529614	.	G	A`
```
diff_end = 23525075 - should be 23529502
coding_region_end = 23525126 - should be 23529502
```

```suggestion
      $diff_end = $second_last_exon->start;
      $coding_region_end = $diff_end + 51;
```